app/src/main/java/ca/bc/gov/bchealth/ui/healthrecord/labtest/LabTestDetailAdapter.kt

```diff
@@ -70,7 +71,8 @@ class LabTestDetailAdapter :
                     val pair = getTestResult(labTestDetail.outOfRange, holder.itemView.context)
                     tvDesc2.text = pair.first
                     tvDesc2.setTextColor(pair.second)
-                    tvDesc2.setTypeface(tvDesc1.typeface, Typeface.BOLD)
+                    val typeface = ResourcesCompat.getFont(holder.itemView.context, R.font.bc_sans_bold)
```

Comment:
Why are we setting typeface programitally?

Reply:
Because its the same item used in recycler view for showing both header and list item. Header does not need bold font where as list items need bold font as per the design.